dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -95,24 +95,28 @@ def create_level_with_rubric
 
   test "should get filtered levels with just page param" do
     get :get_filtered_levels, params: {page: 1}
-    assert_equal JSON.parse(@response.body).length, 7
+    assert_equal 7, JSON.parse(@response.body)['levels'].length
+    assert_equal 22, JSON.parse(@response.body)['numPages']
```

Comment:
Out of curiosity, how do we know how many levels will be found for this test? Do we rely on the database being seeded in a certain way?

Reply:
Since we now limit to 150 results and all of the environments have more than 150 levels in them we can rely on there being 22 pages with 7 results per page.